scripts/git/prune_old_tags.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Deletes the input number of tags and corresponding releases, starting with the
```

Comment:
Did you verify that deleting the tags deletes the releases?

Reply:
Yes!  Deleting a tag deletes the corresponding release